library/classes.lisp

```diff
@@ -113,9 +113,9 @@
   (repr :enum)
   (define-type Ord
     "The result of an ordered comparison."
-    LT
-    EQ
-    GT)
+    (LT "Less than")
```

Comment:
I think I'd rather have the string follow the constructor, not be parenthesized with it.

Reply:
(I'm not set on this but it might be worth looking at a few different syntax options.)

Reply:
An lternative that has the the same flavor:

```

(define-type (T :a)
   T1
   (T2 "Literally T2")
   ((T3 "Nullary T3 Constructor"))
   ((T4 "This one takes an arg") :a))

```

Or an alternative of a different flavor altogether

```

(define-type (T :a)
  T1 
  (T2)
  (T3)
 (T4 :a))

(doc T1 "Literally T1" 
        T2 "Literally T2")

(doc T3  "A nullary constructor for T3. We're putting this docstring in a new `doc` evocation"
         T4 "This one takes an argument")

```